scripts/install_software.sh

```diff
@@ -9,11 +9,7 @@
 YB_HOME=/home/ec2-user/yugabyte-db
 
 YB_VERSION=$1
-if [[ "${YB_VERSION}" < 1.3.0.0 ]]; then
-   YB_PACKAGE_URL="https://downloads.yugabyte.com/yugabyte-ce-${YB_VERSION}-linux.tar.gz"
-else
-   YB_PACKAGE_URL="https://downloads.yugabyte.com/yugabyte-${YB_VERSION}-linux.tar.gz"
-fi
+YB_PACKAGE_URL="https://downloads.yugabyte.com/yugabyte-${YB_VERSION}-linux.tar.gz"
```

Comment:
This link allows downloading only 1.3.0.0 version, what if we want to download an older version ?

Reply:
why only 1.3.0.0?. We can override the DBVersion parameter to use a different version. That is how I tested it as well.